bin/import-previous-release-team-keyring

```diff
@@ -9,12 +9,16 @@ KEYS="9554F04D7259F04124DE6B476D5A82AC7E37093B \
       56730D5401028683275BD23C23EFEFE93C4CFFFE \
       7937DFD2AB06298B2293C3187D33FF9D0246406D"
 
-
-SERVERS="ipv4.pool.sks-keyservers.net \
-         pgp.mit.edu"
+SERVERS="ha.pool.sks-keyservers.net
+         hkp://p80.pool.sks-keyservers.net:80 \
+         hkp://ipv4.pool.sks-keyservers.net \
+         keyserver.ubuntu.com
```

Comment:
Do all these strings need to start with `hkp://`?

Reply:
I tried with and without, and yea, looks like it needs it. After testing it, I actually need to add `hkp://` to `ha.pool.sks-keyservers.net` as well. I'll make that change now.

Reply:
I tested the others as-is, and they respond correctly.
`keyserver.ubuntu.com` doesn't need it when you don't specify the port, but also works with hkp specified.
`pgp.mit.edu` also doesn't need it, but also works with hkp specified.

but that looks inconsistent, so I'll just put it back into the `gpg` command below and take the protocol off from each in the list.